docs/design/api-versions.md

```diff
@@ -16,13 +16,21 @@ If we have a valid **v1** `Person`, trying to submit that through the **v2** ARM
 
 ## Proposed Solution
 
-We need to preserve the original API Version of each resource, and use that to create an appropriate resource for ARM.
+**Preserve the API Version**: the original API version used to create the custom resource (CR) will be preserved so that we know which API version to use when interacting with ARM.
+
+**New conversion interfaces**: to avoid having to version-convert an entire resource when we are interested in only its `Spec` or `Status`, we'll define and implement new interfaces to allow direct conversion.
```

Comment:
New interfaces on the `spec` and `status`? I'm 98% sure the answer is yes but might be worth clarifying here how "new interfaces" provides the "avoiding having to version-convert an entire resource" problem.

Reply:
Oh maybe you go into this below

Reply:
I'll add a brief introductory paragraph here to indicate that this section is high level.